bin/oneoff/backfill_data/school_info_id

```diff
@@ -20,7 +20,7 @@ end
 puts "\nPROCESSED SCHOOL_INFO"
 
 puts "\nGETTING USER_IDS"
-user_ids = Pd::Enrollment.with_deleted.select(:user_id).where.not(school_info_id: nil).distinct.pluck(:user_id)
+user_ids = Pd::Enrollment.with_deleted.select(:user_id).where.not(school_info_id: nil).distinct.pluck(:user_id).compact
```

Comment:
Why not `where.not(user_id: nil)` to filter in SQL rather than on in memory here?

Reply:
I can't react it github either, so 👍